layouts/community/75_ansi/spidey3/keymap.c

```diff
@@ -0,0 +1,35 @@
+#include QMK_KEYBOARD_H
+#include "unicode.h"
+#include "spidey3.h"
```

Comment:
I don't think you need the unicode.h include there, if you have it in your spidey3.h.  

Also, you can cheat... if you include the `QMK_KEYBOARD_H` in your `spidey3.h` file, you should only need that include here.    Either works just fine, but it's a bit of cleverness that may make things simpler.

Reply:
Added unicode.h include to spidey3.h, and removed it here.